subsys/bluetooth/controller/Kconfig

```
@@ -469,6 +469,12 @@ config BT_CTLR_ADV_SET
 	help
 	  Maximum supported advertising sets.
 
+config BT_CTLR_ADV_DATA_LEN_MAX
+	int "Maximum Advertising Data Length"
+	range 31 1650
```

```suggestion
	range 31 1650
	default 251
```
Or some other suitable value

---

done